master/reference/calicoctl/resources/workloadendpoint.md

```diff
@@ -35,7 +36,7 @@ metadata:
   namespace: default
   labels:
     app: frontend
-    calico/k8s_ns: default
+    calico/kns: default
```

Comment:
This one should actually be `projectcalico.org/namespace: default`

Reply:
Ohhh, OK, that's good to know. Hm